src/components/missions/MissionItem.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { joinMission, leaveMission } from '../../redux/missions/missions';
+import style from './missionItem.module.css';
+
+const MissionItem = ({
+  mission_id, mission_name, mission_description, mission_reserved,
+}) => {
+  const dispatch = useDispatch();
+  const handleClick = (id, action) => {
+    if (action === 'join') {
+      dispatch(joinMission(id));
+    } else {
+      dispatch(leaveMission(id));
+    }
+  };
+  return (
+    <tr key={mission_id}>
+      <td>{mission_name}</td>
+      <td>{mission_description}</td>
+      {mission_reserved ? <td className="badge rounded-pill bg-primary text-light m-2">ACTIVE MEMBER</td> : <td className="badge rounded-pill bg-secondary text-light m-2">NOT A MEMBER</td>}
+      <td className={style.bttnwidth}>{mission_reserved ? <button className="btn btn-outline-danger m-2" type="button" onClick={() => handleClick(mission_id, 'leave')}>{mission_reserved ? 'LEAVE MISSION' : 'JOIN NOW' }</button> : <button className="btn btn-outline-secondary m-2" type="button" onClick={() => handleClick(mission_id, 'join')}>{mission_reserved ? 'LEAVE MISSION' : 'JOIN NOW' }</button> }</td>
```

Comment:
- [ ] Rockets/Dragons and Missions should use the React conditional rendering syntax. (This is already being used in your rocket and profile pages but is missing in your missions one, please correct it to comply with the project requirements):

```javascript
{rocket.reserved && ( 
    // render Cancel Rocket button
)}
```

Reply:
Done ✔